arch/arm/boot/dts/overlays/imx708-overlay.dts

```diff
@@ -95,6 +111,7 @@
 		vcm = <&vcm_node>, "status",
 		      <0>, "=4";
 		link-frequency = <&cam_endpoint>,"link-frequencies#0";
+		4lane = <0>, "+201+202";
```

Comment:
It looks like there could be a confusing interaction between `link-frequency` and `4lane` overrides, which _both_ try to change `<&cam_endpoint>/link-frequencies`. The actual frequency may depend on the order in which the dtparams are given.

If that's the case, is there a cleaner way to do it that would always give priority to the `link-frequency`?

Reply:
Sadly there isn't a way to set priorities on DT overrides.

What could be done is to add a fragment 203 which sets the link frequency for 2 lane mode. `link-frequency` then updates both 202 and 203, and `4lane` disables 203 as well as enabling 201 and 202. At least that should give a consistent behaviour.